pipeline/main.py

```diff
@@ -0,0 +1,19 @@
+if __name__ == "__main__":
+    """Run the pipeline process"""
+
+    import os
+
+    if "DATABRICKS_RUNTIME_VERSION" in os.environ:
+        import sys
+
+        sys.path.append("/dbfs/") # add /dbfs/ to path so that import statements works on databricks
+
+    from pipeline.jobs.fill_missing_months import (
```

Comment:
Best practice: Import should be on the top of the file. Also remove parentheses.